### PR TITLE
Work around https://github.com/jline/jline2/issues/282

### PR DESCRIPTION
**Pull request by chrisseaton**

### CODE REVIEW

File: test/truffle/compiler/can-we-fold-yet.sh [hunk 1]
```diff
@@ -2,7 +2,8 @@
 
 source test/truffle/common.sh.inc
 
-jt ruby --graal -J-Dgraal.TruffleIterativePartialEscape=true doc/samples/can-we-fold-yet.rb < test/truffle/compiler/can-we-fold-yet/input.txt > actual.txt
+# tr works around https://github.com/jline/jline2/issues/282
+jt ruby --graal -J-Dgraal.TruffleIterativePartialEscape=true doc/samples/can-we-fold-yet.rb < test/truffle/compiler/can-we-fold-yet/input.txt | tr -d '\15\32' > actual.txt
```
Comment (eregon) [inline test/truffle/compiler/can-we-fold-yet.sh]:
```bash
tr -d '\r'
```
works for me locally, and it seems less strange than this magic two numbers.

Comment (eregon) [inline test/truffle/compiler/can-we-fold-yet.sh]: Both Linux and macOS man pages mention `\r`

Comment (chrisseaton) [inline test/truffle/compiler/can-we-fold-yet.sh]: > works for me locally

But you didn't see the problem in the first place did you?

Comment (eregon) [inline test/truffle/compiler/can-we-fold-yet.sh]: I tried with:
```bash
ruby -e 'print "\r\n"' | tr -d '\r' | ruby -e 'p STDIN.read'
```

**Pull request merged**